pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -331,19 +332,29 @@ func createPipelineSpec(tool string, commitTime int64, jbsConfig *v1alpha12.JBSC
 	df := "FROM " + extractParam(PipelineRequestProcessorImage, paramValues) + " AS build-request-processor" +
 		"\nFROM " + strings.ReplaceAll(extractParam(PipelineRequestProcessorImage, paramValues), "hacbs-jvm-build-request-processor", "hacbs-jvm-cache") + " AS cache" +
 		"\nFROM " + extractParam(PipelineImage, paramValues) +
-		"\nUSER 0 " +
+		"\nUSER 0" +
+		"\nWORKDIR /root" +
 		"\nENV CACHE_URL=" + doSubstitution("$(params."+PipelineCacheUrl+")", paramValues, commitTime, buildRepos) +
-		"\nCOPY --from=build-request-processor /deployments/ /root/build-request-processor" +
-		"\nCOPY --from=build-request-processor /lib/jvm/jre-17 /root/system-java" +
-		"\nCOPY --from=cache /deployments/ /root/cache" +
-		"\nRUN mkdir -p /root/workspace && mkdir -p /root/settings && microdnf install vim" +
+		"\nENV BUILD_POLICY_DEFAULT_STORE_LIST=central,redhat,jboss,gradleplugins,confluent,gradle,eclipselink,jitpack,jsweet,jenkins,spring-plugins,dokkadev,ajoberstar,googleandroid,kotlinnative14linux,jcs,kotlin-bootstrap,kotlin-kotlin-dependencies" +
+		"\nRUN mkdir -p /root/project /root/software/settings && microdnf install vim curl procps-ng bash-completion" +
+		"\nRUN wget -qO- https://github.com/swsnr/mdcat/releases/download/mdcat-2.0.3/mdcat-2.0.3-x86_64-unknown-linux-musl.tar.gz | tar -xz -C /opt/" +
```

Comment:
I don't really know if this is worth it, vs just displaying the markdown. At the very least it should verify the sha of what it has downloaded.

Reply:
I agree there are pros/cons - I was trying to make it a nicer 'user experience' and being able to highlight to the user what they should do. I do agree about the SHA though.